# Description: Close the directory handle when we're done with it, so that we aren't leaking file descriptors.
# Ubuntu: https://launchpad.net/bugs/276918
# Upstream: https://bugs.freedesktop.org/show_bug.cgi?id=17893
Index: consolekit-0.2.10/src/ck-run-programs.c
===================================================================
--- consolekit-0.2.10.orig/src/ck-run-programs.c
+++ consolekit-0.2.10/src/ck-run-programs.c
@@ -225,6 +225,7 @@
         out_loop:
                 g_free (child_argv[0]);
         }
+        g_dir_close (dir);
 out:
         g_strfreev (env_for_child);
 }
